### PR TITLE
feat(chore): release 4.3.0 #000

### PR DESCRIPTION
**Pull request by ZurabDev**
### Features
- feat(doc): add support for get inputs and outputs from hostDirectives #1710
- feat(helpers): new method prizmInvertObject to invert keys and values for json object
- feat(doc/icons): added info about enum name to copied values #1648
- feat(chore/commitlint): support slash in checker commit messages scope
- feat(doc/paginator): added example with i18n to change texts and tests #1605
- feat(i18n): added access to public api internal dictionaries
- feat(chore): improved checker for commit message with our scopes, types, and issues number #1622
- feat(components/input-layput): add hint for input clear button #1515
- feat(documentation/input-number): update input type for step, min, max values in input number live demo #1500
- feat(documentation/slider): update input type for step, min, max values in slider live demo #1500
- feat(component/navigation-menu): added scroll to navigation menu #1585
- feat(components/table): add table size xl #1651
- feat(components/table): add five lines support for table head
- feat(components/chips): css variable for chips height added #1641
- feat(component/cron): added english localization. #1480, attention, PrizmLanguageCron interface has been changed. Why we do this in ADR: #1617
- feat(components/tooltip): tooltip close on click in showed dropdown #1704
- feat(doc/dialog): add examples how to use custom dialog #1602
- feat(helpers): new func prizmHasChanges to check changes in SimpleChanges
- feat(ci): added skip library if they does not have any test

### Refactor
- refactor(components/tooltip): migrate to composition api instead of extend
- refactor(components/dialog): removed injection from constructor and now defaultOptions is exported #1602
### Bug fixes
- fix(components/hint): switcher extra empty hint shown #1682
- fix(doc/dialog): add dismissable prop to live demo #1551
- fix(icons/icons-full): added skipped provider for PrizmDestroyService #1652
- fix(components/cron): cron selected working with specified tabs #1494
- fix(components/cron): cron months blinks on hover #1642
- fix(components/input-select): incorrect focus work when opened by chevron click
- fix(components/chips): default chips height changed to match the mockups #1641
- fix(components/multiselect): correct paddings in outer multiselect #1640
- fix(documentation/input-multi-select): page title correction for input multi-select
- fix(component/table): sort icon replaced with button. #1486 #1720
- fix(component/table): header width doesn't change when sorted #1561
- fix(components/table): table head height should be 32px on each table size #1643
- fix(component/table): table default cursour style changed
- fix(documentation/table): filter and edit action buttons in examples replaced with buttons. #1486
- fix(documentation/table): table track by and sort-pagination examples improvement #1655
- fix(components/accordion): accordion hint not shown if title changed #1415
- fix(component/dropdown-host): add initialization \_autoReposition and \_placement in input in dropdown #1596
- fix(components/input-number): right style on disabled mode #1644 #1645
- fix(components/panel): panel instruments cut focus styles #1625
- fix(components/input\*): ng100 error for input and hint #461 #1471 #1272
- fix(components): tests with errors for resize observer #1631
- fix(components/input-layout-date-range): fix error with ng100 #492
- fix(components/hint): ng100 error on first time appear #1091
- fix(components/hint): hint sometimes does not hide #1658
- fix(components/scrollbar): theme switching in scrollbar works incorrect #1657
- fix: changed content width in toast #1433

### Checklist:

- [x] Замените NEW_VERSION на новую версию в этом PR
- [x] Замените CURRENT_DATE на текущую дату в этом PR
- [x] Добавить в каждый тип ваши изменения в этом PR
- [x] Удалите не нужные группы изменений для этого PR (Features, Bug Fixes, Breacking Changes, Refactor)
- [x] Добавить изменения в changelog о новой версии
- [x] Изменить версии в CI для деплоя (пока не автоматизировали)
  - [x] .github/workflows/beta-publish-ng17.yml
  - [x] .github/workflows/main-publish-ng17.yml
  - [x] .github/workflows/pre-release-publish-ng17.yml
- [x] Изменить версии в доке на новую
  - [x] apps/doc/src/app/version-manager/current.const.ts
  - [x] apps/doc/src/app/version-manager/current.const.ts.ng17
- [x] Изменить версии в соответствующем массиве
  - apps/doc/src/app/version-manager/versions.constants.ts
- [x] Изменить версию в root package файле
  - [x] package.json
  - [x] package.json.ng17
- [x] Изменить версию package файлов в каждой библиотеке (на данный момент это 13 библиотек)
  - [x] libs/ast/package.json
  - [x] libs/ast/package.json.ng17
  - [x] libs/charts/base/package.json
  - [x] libs/charts/base/package.json.ng17
  - [x] libs/components/package.json
  - [x] libs/components/package.json.ng17
  - [x] libs/core/package.json
  - [x] libs/core/package.json.ng17
  - [x] libs/helpers/package.json
  - [x] libs/helpers/package.json.ng17
  - [x] libs/i18n/package.json
  - [x] libs/i18n/package.json.ng17
  - [x] libs/icons/base/package.json
  - [x] libs/icons/base/package.json.ng17
  - [x] libs/icons/flags/package.json
  - [x] libs/icons/flags/package.json.ng17
  - [x] libs/icons/loader/package.json
  - [x] libs/nxmv/package.json
  - [x] libs/nxmv/package.json.ng17
  - [x] libs/plugin/package.json
  - [x] libs/plugin/package.json.ng17
  - [x] libs/schematics/package.json
  - [x] libs/schematics/package.json.ng17
  - [x] libs/theme/package.json
  - [x] libs/theme/package.json.ng17
- [x] Составить readMe файл с описанием изменений на русском

### CODE REVIEW

File: apps/doc/src/app/about-prizm/changelog/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,69 @@
...
+- fix(components/hint): switcher extra empty hint shown #1682 #1686
+- fix: add initialization \_autoReposition and \_placement in input in dropdown#1596 #1681
+- fix(components/input-number): right style on disabled mode #1644 #1645
+- fix(components/input-number): remove inner hint
+- fix(components/panel): panel instruments cut focus styles #1625
+- fix(components/input\*): ng100 error for input and hint #461 #1471 #1272
+- fix(components): tests with errors for resize observer #1631
+- fix(components/input-layout-date-range): fix error with ng100 #492
+- fix(components/hint): ng100 error on first time appear #1091
+- feat(helpers): new func prizmHasChanges to check changes in SimpleChanges
```
Comment (ickisIckis) [inline apps/doc/src/app/about-prizm/changelog/CHANGELOG.md:62]: стоит переместить в раздел feature

Comment (ZurabDev) [reply apps/doc/src/app/about-prizm/changelog/CHANGELOG.md]: Готово

**Review by ickisIckis — APPROVED** [review]
LGTM

File: apps/doc/src/app/about-prizm/changelog/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,69 @@
...
+
+- refactor(components/tooltip): migrate to composition api instead of extend
+- refactor(components/dialog): removed injection from constructor and now defaultOptions is exported #1602
+
+### Bug fixes
+
+- fix(components/hint): switcher extra empty hint shown #1682
+- fix(doc/dialog): add dismissable prop to live demo #1551
+- fix(icons/icons-full): added skipped provider for PrizmDestroyService #1652
+- fix(components/table): ghost button in table should have no hover state background color style #1486
```
Comment (ickisIckis) [inline apps/doc/src/app/about-prizm/changelog/CHANGELOG.md:39]:
```suggestion
- fix(components/table): ghost button in table should have no hover state background color style #1486
```

File: apps/doc/src/app/about-prizm/changelog/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,69 @@
...
+- fix(components/tooltip): tooltip close on click in showed dropdown #1704
```
Comment (ickisIckis) [inline apps/doc/src/app/about-prizm/changelog/CHANGELOG.md:40]:
```suggestion
```

File: apps/doc/src/app/about-prizm/changelog/CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,68 @@
...
+- fix(components/table): table head height should be 32px on each table size #1643
+- fix(component/table): table default cursour style changed
+- fix(documentation/table): filter and edit action buttons in examples replaced with buttons. #1486
+- fix(documentation/table): table track by and sort-pagination examples improvement #1655
+- fix(components/accordion): accordion hint not shown if title changed #1415
+- fix(component/dropdown-host): add initialization \_autoReposition and \_placement in input in dropdown #1596
+- fix(components/hint): switcher extra empty hint shown #1682 #1686
+- fix: add initialization \_autoReposition and \_placement in input in dropdown#1596 #1681
+- fix(components/input-number): right style on disabled mode #1644 #1645
+- fix(components/input-number): remove inner hint
```
Comment (ickisIckis) [inline apps/doc/src/app/about-prizm/changelog/CHANGELOG.md:57]:
```suggestion
```

File: apps/doc/src/app/about-prizm/changelog/CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,67 @@
...
+
+- refactor(components/tooltip): migrate to composition api instead of extend
+- refactor(components/dialog): removed injection from constructor and now defaultOptions is exported #1602
+
+### Bug fixes
+
+- fix(components/hint): switcher extra empty hint shown #1682
+- fix(doc/dialog): add dismissable prop to live demo #1551
+- fix(icons/icons-full): added skipped provider for PrizmDestroyService #1652
+- fix(components/table): ghost button in table should have no hover state background color style #1486
```
Comment (ickisIckis) [inline apps/doc/src/app/about-prizm/changelog/CHANGELOG.md:39]:
```suggestion
```

**Comment by alexhawkins94** [comment]
Input Password
При disabled = true возможен клик по кнопке "глаза" в инпуте и установка фокуса на него 
chrome_lEGv7rfy3B
chrome_lEGv7rfy3B

для 3х актуально тоже

**Comment by alexhawkins94** [comment]
Contained tabs.
Рамка фокуса на табе сдвигает соседний таб
chrome_tSoP6MWE9c

**Comment by alexhawkins94** [comment]
Cron
Установить бессрочно = false
Установить время начала периода
Сделать клик на инпут выбора окончания периода
ФР: появляется ошибка в консоли  Cannot destructure property 'year' of 'object null' as it is null.
chrome_wjoudM72fB

**Comment by alexhawkins94** [comment]
Input, basic example
Сделать расположение хинта справа от кнопки по аналогии с хинтом возле индикатора в инпуте. 

chrome_xfwG9lfQxn
chrome_DZUTNtXqor

**Comment by alexhawkins94** [comment]
Cron
Переключение вкладок происходит с задержкой
Выполняю переключаю в свитчере секунды-минуты, на экранчике крона подгрузка нужной вкладки происходит с видимой задержкой
chrome_ChaGNh4RyM

**Comment by alexhawkins94** [comment]
при обновлении страницы крона - в нем не подгружается содержимое вкладки
chrome_fevvCR36co

File: libs/components/src/lib/components/cron/cron.component.html [hunk 6]
```diff
@@ -109,7 +109,7 @@
       </div>
     </div>
   </div>
-
+  <div>mz:{{ selectedSwitcherIdx }}</div>
```
Comment (ickisIckis) [inline libs/components/src/lib/components/cron/cron.component.html:112]:
```suggestion
```

**Pull request merged**